File: src/main/java/org/polyfrost/animatium_legacy/hooks/DroppedItemHook.java
package org.polyfrost.animatium_legacy.hooks;

import org.polyfrost.animatium_legacy.Animatium;

public final class DroppedItemHook {
    public static boolean isItemDropped;

    public static boolean isItemPhysicsAndEntityDropped() {
        return Animatium.isItemPhysics && isItemDropped;
    }
}

File: src/main/kotlin/org/polyfrost/animatium_legacy/hooks/DroppedItemHook.kt
package org.polyfrost.animatium_legacy.hooks

import org.polyfrost.animatium_legacy.Animatium

object DroppedItemHook {
    @JvmStatic
    var isItemDropped = false

    @JvmStatic
    fun isItemPhysicsAndEntityDropped() = Animatium.isItemPhysics && isItemDropped
}